pkg/appmanager/appManager_test.go

```diff
@@ -583,7 +587,7 @@ var _ = Describe("AppManager Tests", func() {
 			}
 			appMgr := NewManager(&Params{ConfigWriter: mw})
 			Expect(func() { appMgr.outputConfig() }).ToNot(Panic())
-			Expect(mw.WrittenTimes).To(Equal(1))
+			Expect(mw.WrittenTimes).To(Equal(0))
```

Comment:
Please dont modifiy existing unit tests unless there is a certain need. Please share your comments supporting this change.

Reply:
Since we have created a fork between cccl and as3 execution the test cases for virtual server need not fail every time since the execution depends upon the agent flag now.
Since the syncVirtualServer func call the outconfig func hence it is not expecting it to fail every time. Therefore it was expecting the arguments as below.
    Expected
        <int>: 0
    to equal
        <int>: 1
We have maintained the rest of the test cases as it is by adding m.appMgr.agent = "cccl" .